native/app/actions.js

```diff
@@ -100,16 +101,16 @@ export function updateNote(id, content, lastModified) {
   return { type: UPDATE_NOTE, id, content, lastModified };
 }
 
-export function deleteNote(id, origin) {
-  trackEvent('new-note', {
+export function deleteNotes(ids = [], origin) {
+  trackEvent('delete-note', {
```

Comment:
is there an inconsistency introduced here? does the web extension send this event for 1 note, and the mobile app will send a single event even if multiple notes deleted?

Reply:
Great point 🤔. It should definitely send one request per note deleted. 👍